package.json

```diff
@@ -17,13 +20,14 @@
   },
   "homepage": "https://github.com/tidepool-org/nightwatchtests#readme",
   "dependencies": {
+    "dayjs": "^1.9.1",
```

Comment:
Just a recommendation, but as these tests mature, you _likely_ will want to lock down the version numbers of all of your npm packages (removing the leading caret `^` will do)

Reply:
This is good to know! Thank you! Will implement now.